sdks/universal-router-sdk/src/swapRouter.ts

```diff
@@ -43,6 +63,45 @@ export abstract class SwapRouter {
     })
   }
 
+  /**
+   * Builds the call parameters for a migration from a V3 position to a V4 position.
+   * Some requirements of the parameters:
+   *   - v3RemoveLiquidityOptions.collectOptions.recipient must equal v4PositionManager
+   *   - v3RemoveLiquidityOptions.liquidityPercentage must be 100%
+   *   - input pool and output pool must have the same tokens
+   *   - V3 NFT must be approved, or valid inputV3NFTPermit must be provided with UR as spender
+   */
+  public static migrateV3ToV4CallParameters(options: MigrateV3ToV4Options): MethodParameters {
```

Comment:
shoudl it check that the `migrate` param added to v4-sdk is true?

Reply:
it should imo +